README.md

```diff
@@ -12,7 +12,7 @@
 Add solidus_braintree to your Gemfile:
 
 ```ruby
-gem 'solidus_braintree', github: 'solidusio/solidus_braintree', branch: :master
+gem 'solidus_braintree'
```

Comment:
I think we should also explain to people what to do if they want to use the previous version of the gem (`solidus_braintree`). Is that stated anywhere at the moment?

Reply:
Probably makes sense to add information about the previous version in the README. I'll see what I can do.

Reply:
> Yes, it's not advised to upgrade solidus_braintree from v4 to v5, rather it's intended for new stores and migrations from solidus_paypal_braintree.

Added a TODO for that in the description.